Cannot install monika

**Describe the bug**
There is a report that monika cannot be installed using npm but ok with yarn.

**Screenshots**
<img width="801" alt="Screen Shot 2021-06-08 at 09 33 36" src="https://user-images.githubusercontent.com/5974862/121321625-78fb0380-c938-11eb-9782-d11368ca7cc7.png">


**System:**
Distributor ID: Ubuntu
Description: Ubuntu 18.04.5 LTS
Release: 18.04
Codename: bionic
node v12.14.0
npm 6.13.4

seems like permission problem. when i run with sudo, i get different error.

Now the problem is when installing node-pre-gyp

<img width="1440" alt="Screen Shot 2021-06-22 at 13 48 19" src="https://user-images.githubusercontent.com/5974862/122877899-50740000-d361-11eb-856e-18c997c06255.png">


npm WARN deprecated node-pre-gyp@0.11.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported

> @hyperjumptech/monika@1.3.7 preinstall /usr/local/lib/node_modules/@hyperjumptech/monika
> node scripts/preinstall.js

/usr/local/bin/monika -> /usr/local/lib/node_modules/@hyperjumptech/monika/bin/run

> sqlite3@5.0.2 install /usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/sqlite3
> node-pre-gyp install --fallback-to-build

node-pre-gyp WARN Using request for node-pre-gyp https download 
node-pre-gyp WARN Pre-built binaries not installable for sqlite3@5.0.2 and node@12.14.0 (node-v72 ABI, glibc) (falling back to source compile with node-gyp) 
node-pre-gyp WARN Hit error EACCES: permission denied, mkdir '/usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/sqlite3/lib/binding' 
gyp WARN EACCES current user ("nobody") does not have permission to access the dev dir "/root/.cache/node-gyp/12.14.0"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/sqlite3/.node-gyp"
gyp WARN install got an error, rolling back install
gyp WARN install got an error, rolling back install
gyp ERR! configure error 
gyp ERR! stack Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/sqlite3/.node-gyp'
gyp ERR! System Linux 5.4.0-1049-aws
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "--fallback-to-build" "--module=/usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/sqlite3/lib/binding/napi-v3-linux-x64/node_sqlite3.node" "--module_name=node_sqlite3" "--module_path=/usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/sqlite3/lib/binding/napi-v3-linux-x64" "--napi_version=5" "--node_abi_napi=napi" "--napi_build_version=3" "--node_napi_label=napi-v3"
gyp ERR! cwd /usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/sqlite3
gyp ERR! node -v v12.14.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/sqlite3/lib/binding/napi-v3-linux-x64/node_sqlite3.node --module_name=node_sqlite3 --module_path=/usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/sqlite3/lib/binding/napi-v3-linux-x64 --napi_version=5 --node_abi_napi=napi --napi_build_version=3 --node_napi_label=napi-v3' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:210:5)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:1021:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
node-pre-gyp ERR! System Linux 5.4.0-1049-aws
node-pre-gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/sqlite3
node-pre-gyp ERR! node -v v12.14.0
node-pre-gyp ERR! node-pre-gyp -v v0.11.0
node-pre-gyp ERR! not ok 
Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/sqlite3/lib/binding/napi-v3-linux-x64/node_sqlite3.node --module_name=node_sqlite3 --module_path=/usr/local/lib/node_modules/@hyperjumptech/monika/node_modules/sqlite3/lib/binding/napi-v3-linux-x64 --napi_version=5 --node_abi_napi=napi --napi_build_version=3 --node_napi_label=napi-v3' (1)
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.3.2 (node_modules/@hyperjumptech/monika/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sqlite3@5.0.2 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sqlite3@5.0.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-06-22T06_47_10_805Z-debug.log

--- a/docs/src/pages/installation.md
+++ b/docs/src/pages/installation.md
@@ -43,3 +43,11 @@ monika -h
 ```
 
 See the [CLI Options](https://hyperjumptech.github.io/monika/guides/cli-options) guide to see all the different options possible.
+
+## Troubleshoot
+
+If you have problem installing using `npm`, please check this [issue #216](https://github.com/hyperjumptech/monika/issues/216). It might be related to permission issue. In most cases, it can be solved by running
+
+```
+sudo npm i -g @hyperjumptech/monika --unsafe-perm=true --allow-root
+```

